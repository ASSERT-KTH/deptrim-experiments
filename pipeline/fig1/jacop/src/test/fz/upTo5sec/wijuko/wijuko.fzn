array [1..2] of int: X_INTRODUCED_15 = [1,1];
array [1..3] of int: X_INTRODUCED_21 = [1,1,-1];
array [1..2] of int: X_INTRODUCED_30 = [1,-1];
var 1..9: X_INTRODUCED_0;
var 1..9: X_INTRODUCED_1;
var 1..9: X_INTRODUCED_2;
var 1..9: X_INTRODUCED_3;
var 1..9: X_INTRODUCED_4;
var 1..9: X_INTRODUCED_5;
var 1..9: X_INTRODUCED_6;
var 1..9: X_INTRODUCED_7;
var 1..9: X_INTRODUCED_8;
var 2..18: X_INTRODUCED_9:: is_defined_var;
var 2..18: X_INTRODUCED_10:: is_defined_var;
var 2..18: X_INTRODUCED_11:: is_defined_var;
var 2..18: X_INTRODUCED_12:: is_defined_var;
var 2..18: X_INTRODUCED_13:: is_defined_var;
array [1..9] of var int: x:: output_array([1..3,1..3]) = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8];
constraint int_lin_eq(X_INTRODUCED_15,[X_INTRODUCED_0,X_INTRODUCED_3],12);
constraint int_lin_eq(X_INTRODUCED_15,[X_INTRODUCED_1,X_INTRODUCED_4],8);
constraint int_lin_eq(X_INTRODUCED_15,[X_INTRODUCED_2,X_INTRODUCED_5],13);
constraint int_lin_eq(X_INTRODUCED_15,[X_INTRODUCED_3,X_INTRODUCED_6],7);
constraint int_lin_eq(X_INTRODUCED_21,[X_INTRODUCED_4,X_INTRODUCED_7,X_INTRODUCED_11],0):: defines_var(X_INTRODUCED_11);
constraint int_lin_eq(X_INTRODUCED_15,[X_INTRODUCED_5,X_INTRODUCED_8],15);
constraint int_lin_eq(X_INTRODUCED_21,[X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_13],0):: defines_var(X_INTRODUCED_13);
constraint int_lin_eq(X_INTRODUCED_15,[X_INTRODUCED_1,X_INTRODUCED_2],7);
constraint int_lin_eq(X_INTRODUCED_21,[X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_12],0):: defines_var(X_INTRODUCED_12);
constraint int_lin_eq(X_INTRODUCED_15,[X_INTRODUCED_4,X_INTRODUCED_5],14);
constraint int_lin_eq(X_INTRODUCED_21,[X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_10],0):: defines_var(X_INTRODUCED_10);
constraint int_lin_eq(X_INTRODUCED_21,[X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9],0):: defines_var(X_INTRODUCED_9);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_0,X_INTRODUCED_1],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_0,X_INTRODUCED_2],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_0,X_INTRODUCED_3],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_0,X_INTRODUCED_4],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_0,X_INTRODUCED_5],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_0,X_INTRODUCED_6],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_0,X_INTRODUCED_7],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_0,X_INTRODUCED_8],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_1,X_INTRODUCED_2],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_1,X_INTRODUCED_3],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_1,X_INTRODUCED_4],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_1,X_INTRODUCED_5],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_1,X_INTRODUCED_6],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_1,X_INTRODUCED_7],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_1,X_INTRODUCED_8],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_2,X_INTRODUCED_3],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_2,X_INTRODUCED_4],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_2,X_INTRODUCED_5],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_2,X_INTRODUCED_6],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_2,X_INTRODUCED_7],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_2,X_INTRODUCED_8],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_3,X_INTRODUCED_4],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_3,X_INTRODUCED_5],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_3,X_INTRODUCED_6],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_3,X_INTRODUCED_7],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_3,X_INTRODUCED_8],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_4,X_INTRODUCED_5],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_4,X_INTRODUCED_6],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_4,X_INTRODUCED_7],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_4,X_INTRODUCED_8],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_5,X_INTRODUCED_6],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_5,X_INTRODUCED_7],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_5,X_INTRODUCED_8],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_6,X_INTRODUCED_7],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_6,X_INTRODUCED_8],0);
constraint int_lin_ne(X_INTRODUCED_30,[X_INTRODUCED_7,X_INTRODUCED_8],0);
solve  satisfy;
