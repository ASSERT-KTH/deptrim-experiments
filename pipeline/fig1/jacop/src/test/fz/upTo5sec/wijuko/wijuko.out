x = array2d(1..3, 1..3, [9, 2, 5, 3, 6, 8, 4, 1, 7]);
----------